package.json

```diff
@@ -8,7 +8,7 @@
     "xcuitest",
     "xctest"
   ],
-  "version": "3.2.0",
+  "version": "3.2.1-beta.1",
```

Comment:
Generally best not to bump versions in PRs doing other stuff.

Reply:
Yeah sorry, I needed to publish a beta just to see if this would actually work as expected. It does! I can undo it now.